waku/nwaku.go

```diff
@@ -424,6 +424,13 @@ func (w *Waku) DialPeer(address multiaddr.Multiaddr) error {
 	return w.node.Connect(ctx, address)
 }
 
+// TODO: change pubsub topic to shard notation everywhere
```

Comment:
Left this TODO because currently we use pubsub instead of shards all over the place, but at some point we have to change that